src/uct/tcp/tcp.h

```diff
@@ -75,6 +52,15 @@ typedef struct uct_tcp_ep uct_tcp_ep_t;
 typedef unsigned (*uct_tcp_ep_progress_t)(uct_tcp_ep_t *ep);
 
 
+/**
+ * TCP Connection Manager state
+ */
+typedef struct uct_tcp_cm_state {
+    const char            *name;                              /* CM state name */
+    const char            *description;                       /* CM state description */
```

Comment:
why do we need both string name and description? one of them should be enough

Reply:
this comment was not addressed..

Reply:
sorry, I missed this
the comment was addressed in the last commit